thrift-confignode/src/main/thrift/confignode.thrift

```diff
@@ -0,0 +1,38 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+include "rpc.thrift"
+namespace java org.apache.iotdb.confignode.rpc.thrift
+namespace py iotdb.thrift.confignode
+
+service ConfigIService {
+  rpc.TSStatus setStorageGroup(1:i64 sessionId, 2:string storageGroup);
+
+  rpc.TSStatus deleteStorageGroup(1:i64 sessionId, 2:string storageGroup);
+
+  rpc.TSStatus deleteStorageGroups(1:i64 sessionId, 2:list<string> storageGroups);
+
+  i32 getSchemaPartition(1:i64 sessionId, 2:string device)
+
+  list<i32> getDataPartition(1:i64 sessionId, 2:string device, 3:list<i64> times)
```

Comment:
It is better to add one interface which can get a multi device's data partition.